### ISSUE TITLE
Connection leak in MySQL/PostgreSQL proxy managers

### ISSUE THREAD

**Issue by rnby-mike** [vladimir-bukhtoyarov/bucket4j#246]
Inside `removeProxy(Long key)` method connection is acquired by `datasource.getConnection()` but never closed

The fix may look like this
```java
public void removeProxy(Long key) {
    try (Connection connection = dataSource.getConnection();
         PreparedStatement removeStatement = connection.prepareStatement(removeSqlQuery)) {
        removeStatement.setLong(1, key);
        removeStatement.executeUpdate();
    } catch (SQLException e) {
        throw new BucketExceptions.BucketExecutionException(e);
    }
}
```

**Comment by vladimir-bukhtoyarov** [comment]
@rnby-mike 

Thank you. Fix will be released soon.

**Comment by vladimir-bukhtoyarov** [comment]
@rnby-mike 

Thank you, fix will be released soon.

**Comment by vladimir-bukhtoyarov** [comment]
Released with version 7.4.0
